sprints/sprint_2_ru.html

```diff
@@ -110,6 +110,11 @@ <h2>Incident Response</h2>
       <br>
       <!-- The coverage was built upon <a href="https://github.com/oscd-initiative/Cortex-Analyzers" target="_blank">our fork</a> of repo with TheHive Analyzers and Responders, that we tagged with RE&CT IDs. -->
       Вот почему мы решили сосредоточиться на разработке Модулей Реагирования платформы TheHive. Детальный список задач по данному направлению можно найти <a href="sprint_2_backlog_ru.html" target="_blank">здесь</a>.
+      <br><br>
+      <h2>Результаты</h2>
+      - 242 новых правила <a href="https://github.com/SigmaHQ/sigma/pull/1362">Sigma</a> и 305 правил обновлены; <br>
```

Comment:
please update the russian page accordingly 🙏

Reply:
Done, thx